components/expenses/ExpenseForm.js

```diff
@@ -215,6 +221,15 @@ const refreshPayoutProfile = (formik, payoutProfiles) => {
   formik.setFieldValue('payee', payee);
 };
 
+const HiddenFragment = styled.div`
+  display: ${({ show }) => (show ? 'block' : 'none')};
```

Comment:
Good call to do that, hiding with CSS is way more performant. Having the form displayed in JS when you typed the first character could freeze a bit the interface.

Reply:
Yes, I was also having trouble with Formik since fields are not rendered as you can imagine.